crates/trigger-http/src/lib.rs

```diff
@@ -293,10 +308,16 @@ impl HttpTrigger {
                     }
                 };
                 match res {
-                    Ok(res) => Ok(res),
+                    Ok(mut res) => {
+                        res.extensions_mut().insert(MatchedRoute {
+                            route: raw_route.to_string(),
+                        });
```

Comment:
Minor: a nice pattern for this sort of thing (when you are repeating it):

```rust
impl MatchedRoute {
  fn set_response_extension(resp: &mut Response, route: impl Into<String>) { ... }
  // and / or
  fn with_response_extension(mut resp: Response ,route: impl Into<String>) -> Response { ... }
}

MatchedRoute::set_response_extension(resp, raw_route);
```

Reply:
Nice